actioncable/lib/action_cable/engine.rb

```diff
@@ -26,12 +26,8 @@ class Engine < Rails::Engine # :nodoc:
       options = app.config.action_cable
       options.allowed_request_origins ||= /https?:\/\/localhost:\d+/ if ::Rails.env.development?
 
-      app.paths.add "config/cable", with: "config/cable.yml"
```

Comment:
This is here to make possible for people to change the place where the config is. Let's put it back and all the `app.config.paths["config/cable"].first` as well. There is no reason to remove.